### PR TITLE
basic img preprocessor

### PR DESCRIPTION
**Pull request by Szymon-Stasiak**

### CODE REVIEW

File: app/image_preprocessor.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+def get_image_dataloader(image_dir: str, image_size: int = 64, batch_size: int = 32):
+    transform = transforms.Compose([
+        transforms.Resize((image_size, image_size)),
+        transforms.Lambda(lambda img: img.convert("RGB")),
+        transforms.ToTensor(),
+    ])
+
+    dataset = datasets.ImageFolder(root=image_dir, transform=transform)
+    dataloader = DataLoader(dataset, batch_size=batch_size, shuffle=False)
```
Comment (ATTWGRAT) [inline app/image_preprocessor.py:16]: Lepiej niech będzie do wybrania w argumentach funkcji czy shuffle true czy false

**Review by ATTWGRAT — APPROVED** [review]
Oprócz tej małej rzeczy LGTM
